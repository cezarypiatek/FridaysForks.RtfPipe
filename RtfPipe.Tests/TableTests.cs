using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace RtfPipe.Tests;

[TestClass]
public class TableTests
{
  [TestMethod]
  public void marge_scenario_1() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_2() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_3() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_4() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_5() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_6() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_7() => this.VerifyCurrentScenario();
}
