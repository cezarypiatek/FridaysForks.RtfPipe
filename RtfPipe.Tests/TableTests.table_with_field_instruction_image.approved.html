<div style="font-size:12pt;font-family:&quot;Times New Roman&quot;, serif;">
    <table style="font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0;">
        <colgroup>
            <col style="width:544px;" />
        </colgroup>
        <tr style="text-align:left;">
            <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;vertical-align:top;border:none;padding:16px;">
                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                    <colgroup>
                        <col style="width:512px;" />
                    </colgroup>
                    <tr style="text-align:left;">
                        <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">
                            <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0;">
                                <colgroup>
                                    <col style="width:4.4px;" />
                                    <col style="width:507.6px;" />
                                </colgroup>
                                <tr style="text-align:left;height:16px;">
                                    <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">&nbsp;
                                    </td>
                                </tr>
                                <tr style="text-align:left;">
                                    <td colspan="2" style="font-size:18pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:8px 0 0 0;">
                                        <p style="font-family:Aptos, sans-serif;font-size:12pt;margin:0;">
                                            <img src="https://github.githubassets.com/assets/octocat-logo-805b5c3e249f.png" />
                                        </p>
                                        <h2 style="font-size:18pt;margin:8px 0 0 0;">
                                            <span style="font-weight:normal;">[fancy-repo/front-for-dev] Azure Static Web Apps CI/CD workflow run 
                                            </span>
                                        </h2>
                                    </td>
                                </tr>
                                <tr style="text-align:left;height:16px;">
                                    <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">&nbsp;
                                    </td>
                                </tr>
                            </table>
                        </td>
                    </tr>
                </table>
                <p style="margin:0;">
                    <br />
                </p>
                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0 0 0 0.5px;">
                    <colgroup>
                        <col style="width:511.5px;" />
                    </colgroup>
                    <tr style="text-align:left;">
                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:1px solid #E1E4E8;padding:0;">
                            <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                <colgroup>
                                    <col style="width:509.9px;" />
                                </colgroup>
                                <tr style="text-align:left;">
                                    <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">
                                        <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                            <colgroup>
                                                <col style="width:509.9px;" />
                                            </colgroup>
                                            <tr style="text-align:left;">
                                                <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">
                                                    <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                        <colgroup>
                                                            <col style="width:509.9px;" />
                                                        </colgroup>
                                                        <tr style="text-align:left;">
                                                            <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border-top:none;border-right:none;border-bottom:1px solid #E1E4E8;border-left:none;padding:16px 16px 0 16px;">
                                                                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                    <colgroup>
                                                                        <col style="width:477.9px;" />
                                                                    </colgroup>
                                                                    <tr style="text-align:left;">
                                                                        <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">
                                                                            <p style="font-family:Aptos, sans-serif;margin:0;">
                                                                                <img src="https://github.githubassets.com/assets/actions-1cc0c3ccfe18.png" />
                                                                            </p>
                                                                            <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                <colgroup>
                                                                                    <col style="width:3.3px;" />
                                                                                </colgroup>
                                                                                <tr style="text-align:left;height:12px;">
                                                                                    <td style="font-family:&quot;Segoe UI&quot;, sans-serif;font-size:9pt;border:none;padding:0;">&nbsp;
                                                                                    </td>
                                                                                </tr>
                                                                            </table>
                                                                            <h3 style="font-size:15pt;margin:0;">Azure Static Web Apps CI/CD: Some jobs were not successful
                                                                            </h3>
                                                                            <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                <colgroup>
                                                                                    <col style="width:4.4px;" />
                                                                                </colgroup>
                                                                                <tr style="text-align:left;height:16px;">
                                                                                    <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">&nbsp;
                                                                                    </td>
                                                                                </tr>
                                                                            </table>
                                                                            <p style="margin:0;">
                                                                                <br />
                                                                            </p>
                                                                            <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                <colgroup>
                                                                                    <col style="width:477.9px;" />
                                                                                </colgroup>
                                                                                <tr style="text-align:left;">
                                                                                    <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">
                                                                                        <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                            <colgroup>
                                                                                                <col style="width:477.9px;" />
                                                                                            </colgroup>
                                                                                            <tr style="text-align:left;">
                                                                                                <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">
                                                                                                    <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                                        <colgroup>
                                                                                                            <col style="width:477.9px;" />
                                                                                                        </colgroup>
                                                                                                        <tr style="text-align:left;">
                                                                                                            <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">
                                                                                                                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                                                    <colgroup>
                                                                                                                        <col style="width:162.5px;" />
                                                                                                                    </colgroup>
                                                                                                                    <tr style="text-align:left;">
                                                                                                                        <td style="font-family:Aptos, sans-serif;font-size:12pt;text-align:center;background:#28A745;border:none;padding:1px;">
                                                                                                                            <mark style="color:#FFFFFF;background:#1F883D;">
                                                                                                                                <a style="font-family:&quot;Segoe UI&quot;, sans-serif;" href="_blank">View workflow run
                                                                                                                                </a>
                                                                                                                            </mark> 
                                                                                                                        </td>
                                                                                                                    </tr>
                                                                                                                </table>
                                                                                                            </td>
                                                                                                        </tr>
                                                                                                    </table>
                                                                                                </td>
                                                                                            </tr>
                                                                                        </table>
                                                                                    </td>
                                                                                </tr>
                                                                            </table>
                                                                            <p style="margin:0;">
                                                                                <br />
                                                                            </p>
                                                                            <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                                <colgroup>
                                                                                    <col style="width:8.8px;" />
                                                                                </colgroup>
                                                                                <tr style="text-align:left;height:32px;">
                                                                                    <td style="font-family:&quot;Segoe UI&quot;, sans-serif;font-size:24pt;border:none;padding:0;">&nbsp;
                                                                                    </td>
                                                                                </tr>
                                                                            </table>
                                                                        </td>
                                                                    </tr>
                                                                </table>
                                                            </td>
                                                        </tr>
                                                    </table>
                                                    <p style="margin:0;">
                                                        <br />
                                                    </p>
                                                    <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                        <colgroup>
                                                            <col style="width:509.9px;" />
                                                        </colgroup>
                                                        <tr style="text-align:left;">
                                                            <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border-top:none;border-right:none;border-bottom:1px solid #E1E4E8;border-left:none;padding:0;">
                                                                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                    <colgroup>
                                                                        <col style="width:0.4px;" />
                                                                        <col style="width:40px;" />
                                                                        <col style="width:428.9px;" />
                                                                        <col style="width:40.7px;" />
                                                                    </colgroup>
                                                                    <tr style="text-align:left;">
                                                                        <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">&nbsp;
                                                                        </td>
                                                                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:16px 0 16px 16px;">
                                                                            <img src="https://github.githubassets.com/assets/x-circle-fill-red-153d16960fc8.png" />
                                                                        </td>
                                                                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:16px;">
                                                                            <strong>Azure Static Web Apps CI/CD
                                                                            </strong> / Build and Deploy Job 
                                                                            <br />Failed in 3 minutes 
                                                                        </td>
                                                                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:16px 16px 16px 0;">
                                                                            <img src="https://github.githubassets.com/assets/report-gray-d5f2721544ee.png" />
                                                                            <a style="color:#6A737D;text-decoration:none;" href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ffancy-repo%2Ffront-for-dev%2Factions%2Fruns%2F10044629061&amp;data=05%7C02%7C%7Cf49946794ea84e652c5608dcaa6b1aad%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638572624203759287%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&amp;sdata=mbpy4XrpqKFe2dyMKxEMxEffmlWZO79%2FnJIu3xbWKVI%3D&amp;reserved=0">2 
                                                                            </a>
                                                                        </td>
                                                                    </tr>
                                                                </table>
                                                            </td>
                                                        </tr>
                                                    </table>
                                                    <p style="margin:0;">
                                                        <br />
                                                    </p>
                                                    <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                        <colgroup>
                                                            <col style="width:509.9px;" />
                                                        </colgroup>
                                                        <tr style="text-align:left;">
                                                            <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">
                                                                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                                                                    <colgroup>
                                                                        <col style="width:0.4px;" />
                                                                        <col style="width:40px;" />
                                                                        <col style="width:453.1px;" />
                                                                        <col style="width:16.4px;" />
                                                                    </colgroup>
                                                                    <tr style="text-align:left;">
                                                                        <td style="font-size:12pt;text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:1px;">&nbsp;
                                                                        </td>
                                                                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:16px 0 16px 16px;">
                                                                            <img src="https://github.githubassets.com/assets/skip-gray-7e79a83b13ac.png" />
                                                                        </td>
                                                                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:16px;">
                                                                            <strong>Azure Static Web Apps CI/CD
                                                                            </strong> / Close Pull Request Job 
                                                                            <br />Skipped 
                                                                        </td>
                                                                        <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:16px 16px 16px 0;">&nbsp;
                                                                        </td>
                                                                    </tr>
                                                                </table>
                                                            </td>
                                                        </tr>
                                                    </table>
                                                </td>
                                            </tr>
                                        </table>
                                    </td>
                                </tr>
                            </table>
                        </td>
                    </tr>
                </table>
                <p style="margin:0;">
                    <br />
                </p>
                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                    <colgroup>
                        <col style="width:512px;" />
                    </colgroup>
                    <tr style="text-align:left;">
                        <td style="text-align:center;font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;color:#666666;border:none;padding:1px 1px 11px 1px;">
                            <table style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0;">
                                <colgroup>
                                    <col style="width:4.4px;" />
                                </colgroup>
                                <tr style="text-align:left;height:16px;">
                                    <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">&nbsp;
                                    </td>
                                </tr>
                            </table>
                            <p style="color:#000000;margin:0;">
                                <br />
                            </p>
                            <table style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0;">
                                <colgroup>
                                    <col style="width:4.4px;" />
                                </colgroup>
                                <tr style="text-align:left;height:16px;">
                                    <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">&nbsp;
                                    </td>
                                </tr>
                            </table>
                            <p style="margin:0 0 10px 0;">—
                                <br />You are receiving this because you are subscribed to this thread.
                                <br />
                                <a style="color:#0366D6;" href="https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsettings%2Fnotifications&amp;data=05%7C02%7C%7Cf49946794ea84e652c5608dcaa6b1aad%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638572624203776269%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&amp;sdata=67xU8y6zLTdgl%2FN94zoK%2B4q4eeNngejl7Ib5ocAHnsE%3D&amp;reserved=0">Manage your GitHub Actions notifications
                                </a>
                            </p>
                        </td>
                    </tr>
                </table>
                <p style="margin:0;">
                    <br />
                </p>
                <table style="font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;">
                    <colgroup>
                        <col style="width:512px;" />
                    </colgroup>
                    <tr style="text-align:left;">
                        <td style="text-align:center;font-family:&quot;Segoe UI&quot;, sans-serif;font-size:9pt;color:#6A737D;border:none;padding:1px 1px 11px 1px;">
                            <table style="color:#000000;font-family:&quot;Times New Roman&quot;, serif;font-size:10pt;border-spacing:0;box-sizing:border-box;margin:0;">
                                <colgroup>
                                    <col style="width:4.4px;" />
                                </colgroup>
                                <tr style="text-align:left;height:16px;">
                                    <td style="font-size:12pt;font-family:&quot;Segoe UI&quot;, sans-serif;border:none;padding:0;">&nbsp;
                                    </td>
                                </tr>
                            </table>
                            <p style="margin:0 0 10px 0;">GitHub, Inc. 
                                <span style="font-family:&quot;MS Gothic&quot;, monospace;">・
                                </span>88 Colin P Kelly Jr Street 
                                <span style="font-family:&quot;MS Gothic&quot;, monospace;">・
                                </span>San Francisco, CA 94107
                            </p>
                        </td>
                    </tr>
                </table>
            </td>
        </tr>
    </table>
    <p style="text-align:left;font-family:&quot;Segoe UI&quot;, sans-serif;font-size:11.5pt;color:#24292E;margin:0;">
        <br />
    </p>
    <p style="font-family:Aptos, sans-serif;text-align:left;margin:0;">
        <br />
    </p>
 </div>